### PR TITLE
Student: submit responses: tweak info about Equal Share scale #7557

### LINKED ISSUE #7557: Student: submit responses: tweak info about Equal Share scale
Current:
image

Suggested: 
More info about the 'Equal share' scale
OR 
More info about the _Equal share_ scale
OR 
More info about the `Equal share` scale

---

Current:
image

Suggested:
>`Equal share` is a relative measure of individual contribution to a team task.
For example, in a 3-person team, 'equal share' means a third of the work done.
`Equal share + 10%` means the person did about 10% _more_ than an equal share, `Equal share - 10%` means about 10% _less_ than an equal share, and so on.

### PR DESCRIPTION
**Pull request by adityavhegde**
Fixes #7557 

* The `Equal Share` text has been highlighted using the `<code>` tag in the 2 respective files.
* Some new text was introduced in the modal window as requested.
* `<em>` tag has been used (in place of `<i>` tag) to italicize words in modal window, in order to maintain semantics.

UI Change: 
image

### CODE REVIEW

**Comment by adityavhegde** [comment]
`teammates.test.cases.browsertests.AdminHomePageUiTest` did not change after running tests in god mode. Did I miss something ?

**Comment by HirdayGupta** [comment]
@adityavhegde It would be hard to tell if you've missed something while running god mode without information on how you ran it. However, I would suggest going through [our docs on testing](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#testing) as well as [our docs on godmode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) and check if you've skipped a step. (most prominent step would be to re-confirm if you set the `test.godmode.enabled` property to `true` in `test.properties`. If you don't see this property in this file refer to #7464)

I often find that simply re-running the test after confirming god mode is enabled does the trick. 

Another note: do post screenshots whenever you make UI changes as it gives your reviewer a sense of the changes you have made.

FYI: I am **not** your reviewer but a co-contributor making suggestions.

**Comment by HirdayGupta** [comment]
@adityavhegde did you try re-running god mode?

**Comment by adityavhegde** [comment]
@HirdayGupta not yet, my project setup has fallen apart.

**Comment by HirdayGupta** [comment]
@adityavhegde I see. Raise an issue in the Issue Tracker if you think you might need help mending it. If it is to do the GAE Update introduced in #7608, take a look at the solution by @whipermr5 in #7627. 

If it's anything else, try running `./gradlew resetEclipseDeps` and then refreshing and cleaning your project (if on Eclipse). Take a look out [our troubleshooting docs](https://github.com/TEAMMATES/teammates/blob/master/docs/troubleshooting-guide.md#common-setup-errors-and-solutions).

**Comment by adityavhegde** [comment]
@HirdayGupta thanks a lot. I see, there was some problem with GAE. I am currently using the older version, which was working fin. I don't know what went wrong suddenly. After having spent a day trying to figure, I did a clean install now. Those links would really be useful. Thanks for the heads up.

**Comment by HirdayGupta** [comment]
No worries @adityavhegde here to help :) Do post details whenever there is an issue you need help with.

**Comment by adityavhegde** [comment]
@HirdayGupta 

Your guess was right, re-running with God mode resulted in a successful build. I am tad bit skeptical though. There is this `timeout exception` below, which I do not understand:

image

Any comments ?

**Comment by HirdayGupta** [comment]
@adityavhegde I wouldn't worry about individual timeouts on local test runs. A lot of random things could cause them locally. Travis passing you is what matters. And the tests seem to have passed, well done! 😃

**Comment by HirdayGupta** [comment]
@adityavhegde Forgot to add, keep an eye out for failing local tests related to changes you have made.  That would mean you have to change something.

**Comment by adityavhegde** [comment]
Sure @HirdayGupta  :)

File: src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
             <div class="modal-header">
                 <button type="button" class="close" data-dismiss="modal">&times;</button>
-                <h4 class="modal-title">More info about the 'Equal Share' scale</h4>
+                <h4 class="modal-title">More info about the <code>Equal Share</code> scale</h4>
             </div>
             <div class="modal-body">
-                <p>Equal share is a relative measure of individual contribution to a team.</p>
-                <p>For example, in a 3 person team, 'equal share' means a third of the work done.</p>
+                <p><code>Equal share</code> is a relative measure of individual contribution to a team task.</p>
+                <p>For example, in a 3-person team, <code>equal share</code> means a third of the work done.</p>
```
Comment (whipermr5) [inline src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html]: Suggestion to capitalise this (i.e. `Equal share`) to match the option shown in the dropdown.

Comment (adityavhegde) [inline src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html]: > <p>For example, in a 3-person team, <code>equal share</code> means a third of the work done.</p>
 @whipermr5 Do you mean to change on this line to: <p>For example, in a 3-person team, <code>Equal share</code> means a third of the work done.</p>

Comment (whipermr5) [inline src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html]: Yes :)

Comment (adityavhegde) [inline src/main/resources/feedbackQuestionContribSubmissionFormTemplate.html]: Sure :)

**Comment by adityavhegde** [comment]
@whipermr5 Thank you:) In line with your previous review: I noticed that in the `More info` link, the text prior to changes was `Equal share`. However, I seemed to have changed it to `Equal Share`. Does that look OK to you?

**Comment by whipermr5** [comment]
@adityavhegde I did notice it; I think it's fine as it matches the modal title :)

**Pull request merged**